ui/src/components/Publication/Breakdown/index.tsx

```diff
@@ -9,82 +10,42 @@ type Props = {
-            {/** circle */}
-            <div className="relative mx-auto">
-                <Components.ProgressCircle values={values} width={100} height={100} thickness={20} />
-                <div className="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
-                    <span className="mb-4 block text-center font-montserrat text-5xl font-medium text-teal-500 transition-colors duration-500 dark:text-white-50">
-                        {totalCount}
-                    </span>
-                    <span className="block text-center font-montserrat text-2xl transition-colors duration-500 dark:text-grey-50">
-                        Publications
-                    </span>
-                </div>
-            </div>
-
-            {/** bars */}
-            <div className="grid items-center gap-y-3 lg:gap-y-0">
-                {values.map((value) => {
-                    const valuePercentage = Math.round(Helpers.percentage(value.value, props.publications.length));
-                    const highestPercentage = Math.round(Helpers.percentage(highestValue, props.publications.length));
-
-                    let max = highestPercentage;
-                    if (highestPercentage > 350) max = 350;
-                    if (highestPercentage < 200) max = 200;
-
-                    return (
-                        <div
-                            key={value.title}
-                            className="grid grid-cols-[auto_auto] lg:grid-cols-[minmax(12.5rem_,_max-content)_minmax(6rem_,_max-content)_auto]"
-                        >
-                            <span
-                                key={value.title}
-                                className="font-montserrat transition-colors duration-500 dark:text-grey-100 lg:text-right"
-                            >
-                                {value.title}
-                            </span>
-                            <span
-                                key={value.value}
-                                className="mx-4 text-right font-montserrat transition-colors duration-500 dark:text-grey-100 lg:text-center"
-                            >
-                                ({value.value})
-                            </span>
-                            <Components.ProgressBar
-                                key={value.color}
-                                value={Helpers.percentage(valuePercentage, highestPercentage)}
-                                color={value.color}
-                                width={max}
-                                className="col-span-2 mt-1 h-4 lg:col-span-1 lg:mt-0 lg:h-full"
-                            />
-                        </div>
-                    );
-                })}
-            </div>
+        <div>
+            <Chart.Radar data={data} />
```

Comment:
Do you think that we should keep the chart in for now or remove it based on the talks in the standup? If not, maybe we remove the logic & packages and take it fresh when it comes round?

Reply:
OK sweet, happy to do that yeah